lib/cluster.js

```diff
@@ -127,13 +127,13 @@ function RoundRobinHandle(key, address, port, addressType, fd) {
     this.server.listen(port, address);
   else
     this.server.listen(address);  // UNIX socket path.
-
-  this.server.once('listening', () => {
```

Comment:
ok, this needs the es6 short form so it can access the scoped `this`. I still don't understand the introduction of an intermediate const, it appears to a mechanical source code translation without effect. Are all es6 functions in the code base being assigned to consts? Can you reference the issue that specifies this?